packages/paste-website/src/components/customization-landing-page/LandingPageLayoutUtils.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react';
+import {Box} from '@twilio-paste/box';
+import type {BoxProps} from '@twilio-paste/box';
+import {SITE_CONTENT_MAX_WIDTH} from '../../constants';
+
+export const LandingPageSection: React.FC<BoxProps> = ({children, ...props}) => {
+  return (
+    <Box
+      as="section"
+      element="LANDING_PAGE_SECTION"
+      paddingX={['space90', 'space120', 'space190']}
+      paddingY={['space90', 'space120', 'space200']}
+      position="relative"
+      {...props}
+    >
+      {children}
+    </Box>
+  );
+};
+
+const ContentVariants = {
+  default: {
+    maxWidth: SITE_CONTENT_MAX_WIDTH,
+  },
+  narrow: {
+    maxWidth: 'size90',
+  },
+} as const;
+
+type LandingPageSectionContentProps = {
+  variant: keyof typeof ContentVariants;
+} & BoxProps;
+export const LandingPageSectionContent: React.FC<LandingPageSectionContentProps> = ({
+  children,
+  variant = 'default',
+  ...props
+}) => {
+  return (
+    <Box
+      element="LANDING_PAGE_SECTION_CONTENT"
```

Comment:
Why give these elements?

Reply:
I was actually using them to debug the DOM structure in the browser to see what ended and started where. Kinda useful